com.insynchq.Insync.appdata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.insynchq.Insync</id>
+  <name>Insync</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>Insync</developer_name>
+  <summary>Google Drive, OneDrive and Dropbox syncing</summary>
+  <url type="homepage">https://www.insynchq.com/</url>
+  <description>
+    <p>
+    Manage your Google Drive, OneDrive, and Dropbox files straight from your Desktop
+    </p>
+    <p>
+    With Insync, you can now sync, edit, and share files without opening a million browser tabs
+    </p>
+  </description>
+  <keywords>
+    <keyword>sync</keyword>
+    <keyword>synchronization</keyword>
+  </keywords>
+  <launchable type="desktop-id">com.insynchq.Insync.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/razzeee/com.insynchq.Insync/master/screenshots/welcome.png</image>
```

Comment:
NEVER the tip of a branch.

Reply:
True, needs to be changed after repo creation to target that repo anyway - but I can change this later to ref the commit